common_repo_templates/CONTRIBUTING_template.md

```diff
@@ -16,18 +17,22 @@ All contributions to the repository must be submitted under the terms of the [Ap
 
 ## Certificate of Origin
 
-By contributing to this project, you agree to the Developer Certificate of Origin (DCO). This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution. See the [DCO](DCO) file for details.
+By contributing to this project, you agree to the Developer Certificate of Origin (DCO). This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution. See the [DCO](https://github.com/open-cluster-management/community/blob/main/DCO) file for details.
```

Comment:
for the link change I don't think its necessary because each repo will still have their own DCO file as the community convention but this can work too.

Reply:
I thought that was the point of @juliana-hsu's [post](https://github.com/open-cluster-management/backlog/issues/6715#issuecomment-820552067)--that we can not have it in every repo?

Reply:
We need those files in every repo but they will become pointers themselves. 
see https://github.com/open-cluster-management/community/pull/33
so each repo will have a copy of those files but they will be pointers that seems to be the community convention.

Reply:
Okay, that's not exactly less work, then. Why do we need each of these files in every repo? Providing a link from `CONTRIBUTING.md` doesn't count as providing the file?

Reply:
> Okay, that's not exactly less work,

Maybe initially there is some work for you to copy these files into your repo(s) but if these files ever get updated in the community repo, you don't have to do anything. So that's the "less work" part.

> Why do we need each of these files in every repo? Providing a link from CONTRIBUTING.md doesn't count as providing the file?

I think we are just following convention from other popular open repos. For example,
https://github.com/kubernetes/kubernetes
https://github.com/kubernetes/minikube
etc

they all have these files under their own repo and either populated with content or they are pointers to their respective "community" repo file. As for why they don't have DCO file, I think the Kubernetes contribution need CLA so just DCO is not enough.

Reply:
They have the CLA linked in their `CONTRIBUTING` file.

Reply:
ok lets do that for the DCO file then. Can you delete that DCO file from the template in this PR.
But we will still need to copy the LICENSE, SECURITY and COC files into each repo.

Reply:
I might have jumped the gun a bit, but I think I did what we were discussing. I've updated the CoC and SECURITY template files to link back to `community`. (Sorry, this PR is getting a bit bigger than I thought it would! 🙂 )